django_project/changes/views/version.py

```diff
@@ -175,6 +175,23 @@ def get_context_data(self, **kwargs):
         if project_slug:
             context['project'] = Project.objects.get(slug=project_slug)
 
+        # lets check for specific user permissions here.
+        if self.request.user.is_staff:
+            context['user_can_edit'] = True
+            context['user_can_delete'] = True
+
+        if self.request.user.is_superuser:
+            context['user_can_edit'] = True
+            context['user_can_delete'] = True
+
+        if context['project'].changelog_managers.filter(
+                project__owner=self.request.user.id).exists():
+            context['user_can_edit'] = True
+            context['user_can_delete'] = True
+
+        if self.request.user == context['project'].owner:
+            context['user_can_edit'] = True
+            context['user_can_delete'] = True
```

Comment:
I'm failing to understand, where this could be before? We were using these variables in the template, but not creating these variables in the view? Weird

Reply:
Thanks
Fixed that initialisation.